tests/integration/level_depth/test_search_different_depths.py

```diff
@@ -24,31 +18,41 @@ def test_index_depth_0_search_depth_1():
     def validate_granularity_1(resp):
         assert len(resp.docs) == 3
         for doc in resp.docs:
-            assert doc.granularity == 1
-            assert len(doc.matches) == 1
+            assert doc.granularity == 0
+            assert len(doc.matches) == 3
```

Comment:
why was it 1 and now 3? Not sure it was right in the first place?

Reply:
there was a use of `top_k` beforehand. But since `top_k` us currently buggy, it does not work and I completely removed it.